cmd/istioctl/main.go

```diff
@@ -318,10 +346,12 @@ Example usage:
 func init() {
 	rootCmd.PersistentFlags().StringVarP(&kubeconfig, "kubeconfig", "c", "",
 		"Use a Kubernetes configuration file instead of in-cluster configuration")
-	rootCmd.PersistentFlags().StringVarP(&managerAddr, "managerAddr", "m", "",
-		"Set your Istio manager address")
 	rootCmd.PersistentFlags().StringVarP(&namespace, "namespace", "n", api.NamespaceDefault,
 		"Select a Kubernetes namespace")
+	rootCmd.PersistentFlags().StringVar(&istioNamespace, "istioNamespace", defaultIstioNamespace,
+		"Namespace where Istio system resides")
+	rootCmd.PersistentFlags().IntVar(&apiServerPort, "port", 8081,
+		"Manager API server port")
 
```

Comment:
I agree. Seems like a sensible idea would be to have a generic client for fallback as well as a kube client.